Description: Delay alarm-notifier by 30 seconds. It's not necessary to have in the critical path during boot, and it's quite expensive.
Author: Martin Pitt <martin.pitt@ubuntu.com>
Forwarded: Not yet, depends on https://bugzilla.gnome.org/show_bug.cgi?id=608402

--- a/data/evolution-alarm-notify.desktop.in
+++ b/data/evolution-alarm-notify.desktop.in
@@ -13,3 +13,4 @@
 X-GNOME-Bugzilla-Product=evolution
 X-GNOME-Bugzilla-Component=calendar
 X-GNOME-Bugzilla-Version=3.10.x
+X-GNOME-Autostart-Delay=30
--- a/data/evolution-alarm-notify.desktop.in.in
+++ b/data/evolution-alarm-notify.desktop.in.in
@@ -13,3 +13,4 @@
 X-GNOME-Bugzilla-Product=evolution
 X-GNOME-Bugzilla-Component=calendar
 X-GNOME-Bugzilla-Version=@BASE_VERSION@.x
+X-GNOME-Autostart-Delay=30
